kubernetes-curl-job.sh

```diff
@@ -1,31 +1,31 @@
-#!/bin/sh
+#!/bin/env sh
 
 notifySuccess() {
     curl -X POST -H 'Content-type: application/json'\
     --data "{\"text": "$SUCCESS_MESSAGE" :"\"rocket:\"}" \
-    $WEBHOOK
+    "$WEBHOOK"
```

Comment:
[SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086): Double quote to prevent globbing and word splitting.